CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 - none yet
 
+## [v0.3.21] - 2017-11-02
+### Addes
+- Add .focus() to allow keyboard navigation inside the iframe
+
 ## [v0.3.20] - XXXX-XX-XX
```

Comment:
You have put a date for `v0.3.21` and kept `XXXX-XX-XX` for `v0.3.20`

Reply:
Ok, thx !